dace/codegen/targets/xilinx.py

```diff
@@ -639,7 +639,7 @@ def generate_module(self, sdfg, state, name, subgraph, parameters,
             # Launch the kernel from the host code
             rtl_name = self.rtl_tasklet_name(rtl_tasklet, state, sdfg)
             host_stream.write(
-                f"  auto kernel_{rtl_name} = program.MakeKernel(\"{rtl_name}_top\", {', '.join([name for _, name, p, _ in parameters if not isinstance(p, dt.Stream)])}).ExecuteTaskFork();",
+                f"  auto kernel_{rtl_name} = program.MakeKernel(\"{rtl_name}_top\"{', '.join(['']+[name for _, name, p, _ in parameters if not isinstance(p, dt.Stream)])}).ExecuteTaskFork();",
```

Comment:
Run yapf on the code please!

Reply:
I did. I assume you are referring to the missing spaces around "+"? I added them now but, rather surprisingly, pep8 actually doesn't require spaces around arithmetic operators.

https://hg.python.org/peps/rev/37af28ad2972

Reply:
It does, but it was inside the string format, so I guess it's not smart enough to parse this :-) Thanks!

![pep](https://user-images.githubusercontent.com/2845320/115209981-6e717a00-a0fe-11eb-9a16-0203e3827721.gif)